src/DotLiquid.Tests/ConditionTests.cs

```diff
@@ -695,6 +699,118 @@ public void TestEqualOperatorsWorksOnEnum()
             AssertEvaluatesFalse("enum", "!=", "'Yes'");
         }
 
+        [Test]
+        public void TestBlankObject()
+        {
+            _context = new Context(CultureInfo.InvariantCulture);
+            _context["dict"] = new Dictionary<string, string> { { "abc", "xyz"} };
+            _context["empty_dict"] = new Dictionary<string, string> { };
+            _context["list"] = new List<string> { "abc" };
+            _context["empty_list"] = new List<string> { };
+            _context["arr"] = new string[] { "foo" };
+            _context["empty_arr"] = new string[] {};
+            _context["a_drop"] = new DummyDrop();
+
+            // self check
+            AssertEvaluatesFalse("blank", "==", "blank");

```

Comment:
Daviburg is strict about using named parameters when two or more share the same type.
```suggestion
            AssertEvaluatesFalse(left: "blank", op: "==", right: "blank");
```